docs/UserGuide.md

```diff
@@ -174,19 +174,35 @@ _Details coming soon ..._
 
 ## FAQ
 
-**Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**Q**: How does this app help current librarians?<br>
+**A**: The app helps librarians manage the statuses of books borrowed and borrowers.
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Command summary
 
-Action | Format, Examples
---------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-**Clear** | `clear`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list`
-**Help** | `help`
+### Category: Patron Commands
+| Function                     | Format Of Command                                                           |
+|------------------------------|-----------------------------------------------------------------------------|
+| **Add a new patron**         | `patron add n/<name> s/<id> p/<phone> e/<email> [t/<tag> …]`                |
+| **Edit a current patron**    | `patron edit INDEX [n/<name>] [s/<id>] [p/<phone>] [e/<email>] [t/<tag> …]` |
+| **Delete a current patron**  | `patron delete INDEX`                                                       |
+| **Find a particular patron** | `patron find n/<name keyword> [n/<name keyword> ...]`                       |
+| **List all patrons**         | `patron list`                                                               |
+### Category: Book Commands
+| Function                  | Format Of Command                                                              |
+|---------------------------|--------------------------------------------------------------------------------|
+| **Add a new book**        | `book add n/<name> i/<ISBN> [a/<author> …] [t/<category tag> …]`               |
+| **Edit a current book**   | `book edit INDEX n/<name> i/<ISBN> [a/<author> …] [t/<category tag> …]`        |
+| **Delete a current book** | `book delete INDEX`                                                            |
+| **Borrow a book**         | `borrow b/<index> p/<index> d/<return date>`                                   |
+| **Return a book**         | `return [b/<index>] or [p/<index>]`                                            |
```

Comment:
Not sure if we should merge b/index or p/index as one bubble for both the borrow and return command in the user guide.

There could be ambiguity in [index](https://nusu-my.sharepoint.com/:w:/r/personal/e0665491_u_nus_edu/Documents/Document.docx?d=w71ddd0adbc1c418fa7c2009a26c295fb&csf=1&web=1&e=G8Jed5&nav=eyJjIjo4NjcxNjAwMDB9).

If our user runs the command `return p/<index>`, is the index preferring to the index from the patron list or the book list?

Maybe we can add borrow and return for patron and for book respectively?